counterapp/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
```

Comment:
Currently the node_modules aren't being ignored. I think we need to move the file one level up.